ansible/ansible.cfg

```diff
@@ -14,7 +14,7 @@
 inventory      = /etc/ansible/hosts
 library        = library:library/ixia
 module_utils   = module_utils
-remote_tmp     = $HOME/.ansible/tmp
+remote_tmp     = /tmp/ansible-$USER
```

Comment:
@slutati1536 suggest to have: `/tmp/.ansible-$USER`

Reply:
@nazariig I will test this change in regression tonight, and will update the PR